playbooks/compliance_openscap_remove.yml

```diff
@@ -1,7 +1,10 @@
 ---
-# Placeholder playbook for disabling Compliance
-- hosts: localhost
+# This playbook will take care of all steps required to remove
+# Insights Compliance (OpenSCAP packages)
+- hosts: all
+  become: yes
   tasks:
-    - name: Disable Compliance
-      debug:
-        msg: "Compliance removed"
+    - name: Remove OpenSCAP packages
+      yum:
+        name: openscap, openscap-scanner, scap-security-guide
```

Comment:
I thought the conclusion was to not remove packages when disabling service. Did that change?

Reply:
I wasn't 100% on that so I didn't modify the compliance playbook, but if that's the case we'll need some mechanism to disable openscap that doesn't include yum removing it.

Reply:
Same with the insights-client.. @lphiri Should we only perform the unregister and leave the insights-client package on the host?

Reply:
@Jason-RH - yes, lets leave the packages on  when disabling the services.